experimental/origin-admin/src/pages/marketplace/mutations/CreateListing.js

```diff
@@ -54,11 +54,13 @@ class CreateListing extends Component {
         from: seller ? seller.id : '',
         deposit: 0,
         category: props.listing.category || 'For Sale',
+        subCategory: props.listing.subCategory || '', // TODO: There's no subcategory dropdown in the modal yet
```

Comment:
Without this, the admin modal broke, as there was some update on the `dapp` or `graphql` to require `subCategory`.